osu.Game.Tournament/Components/FumoBeatmapPanel.cs

```diff
@@ -0,0 +1,467 @@
+                    }
+                },
+                trophyIcon = new CircularContainer
+                {
+                    Name = @"Win Circle",
+                    Anchor = Anchor.BottomRight,
+                    Origin = Anchor.BottomRight,
+                    Margin = new MarginPadding { Horizontal = -5, Bottom = -11 },
+                    Size = new Vector2(24),
+                    Masking = true,
+                    Alpha = 0,
+                    Children = new Drawable[]
+                    {
+                        trophyBg = new Box
+                        {
+                            RelativeSizeAxes = Axes.Both,
+                        },
+                        new SpriteIcon
+                        {
+                            Icon = FontAwesome.Solid.Trophy,
+                            Size = new Vector2(14),
+                            Colour = Color4.White,
+                            Anchor = Anchor.Centre,
+                            Origin = Anchor.Centre,
+                        }
+                    }
+                }
+            };
+        }
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            updateBorder();
+            updateState(false);
+        }
+
+        private void matchChanged(ValueChangedEvent<TournamentMatch?> match)
+        {
+            if (match.OldValue != null)
+                match.OldValue.ChessPlacements.CollectionChanged -= onPlacementChanged;
+
+            if (match.NewValue != null)
+                match.NewValue.ChessPlacements.CollectionChanged += onPlacementChanged;
+
+            Scheduler.AddOnce(() => updateState(false));
+        }
+
+        private void onPlacementChanged(object? _, NotifyCollectionChangedEventArgs e)
+            => Scheduler.AddOnce(updateState, e.Action == NotifyCollectionChangedAction.Add);
+
+        private ChessPlacement? lastPlacement;
+        private ScheduledDelegate? scheduledFloatingBoxAnimation;
+
+        private void updateBorder()
+        {
+            if (!IsLoaded)
+                return;
+
+            if (selected)
+            {
+                beatmapInfoContainer.BorderColour = Colour4.White;
+                beatmapInfoContainer.TransformTo(nameof(beatmapInfoContainer.BorderThickness), INNER_BORDER, 500, Easing.OutQuint);
+                backgroundAddition.FlashColour(Color4.White.Opacity(0.9f), 1000, Easing.OutQuint);
+            }
+            else
+            {
+                beatmapInfoContainer.TransformTo(nameof(beatmapInfoContainer.BorderThickness), 0f, 500, Easing.OutQuint);
+            }
+        }
+
+        private void updateState(bool playFullAnimation = true)
+        {
+            // Match unavailable: Clean up
+            if (currentMatch.Value == null)
+            {
+                banPill.FadeOut(300, Easing.OutQuint);
+                trophyIcon.FadeOut(300, Easing.OutQuint);
+                return;
+            }
```

Comment:
```suggestion
            if (currentMatch.Value == null)
            {
                banPill.FadeOut(300, Easing.OutQuint);
                trophyIcon.FadeOut(300, Easing.OutQuint);
                lastPlacement = null;
                return;
            }
```
if necessary, we may need run `FinishTransforms`?

Reply:
Yep, I'm going to add it.